docs/faqs.md

```diff
@@ -260,3 +261,70 @@ In addition, we support only 64-bit builds of the interpreter (amd64/x86_64 arch
 
 There are no guarantees that recipes will work correctly in future Python versions having breaking changes to the interpreter,
  as we don't test all the possible combinations (and probably will never be). Patches are welcomed if problems are found.
+
+## How to package libraries that depend on Intel MKL, IPP or DNN libraries?
+
+There are several popular software libraries provided by Intel:
+
+* [Intel Math Kernel Library (MKL)](https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/onemkl.html)
+* [Intel Integrated Performance Primitives (IPP)](https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/ipp.html)
+* [Intel Deep Neural Networking Library (DNN)](https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/onednn.html)
+
+these Intel libraries are widely used by the various well-known open-source projects (e.g. [OpenCV](https://opencv.org/) or [TensorFlow](https://www.tensorflow.org/)).
+
+unfortunately, these Intel libraries cannot be accepted into ConanCenter due to the several important reasons:
+
+* they are closed-source and commercial products, which we're not going to re-distribute via ConanCenter due to the license restrictions
+* registration on the Intel portal is required in order to dowload the libraries, there are no permanent public direct download links
+* they use graphical installers which are hard to automate within conan recipe
+* they are pre-built binaries, so they will never respect any custom compiler flags specified
```

Comment:
```suggestion
```

This doesn't belong here, it applies to every library that is downloading precompiled binaries. There is probably and entry about it.

Reply:
yes,  I think so